README.md

```diff
@@ -198,13 +198,9 @@ For a complete list of commands, see the
 ## Building the CLI
 ### Developing
 
-Running ``godep save ./ecs-cli/...`` in the `ecs-cli/` directory creates/updates the vendor/
-directory with dependencies. Make sure that you are using go1.6 and that you have the latest godep (`go get -u github.com/tools/godep`).
-
-For developing code, the correct GOPATH can be printed by running the
-`./scripts/shared_env` script.
-
-This can be set as GOPATH on the dev box.
+* Make sure you are using go1.8 (`go version`).
+* Copy the source code (`go get github.com/aws/amazon-ecs-cli`).
+* From `$GOPATH/src/github.com/aws/amazon-ecs-cli`, run `make`.
```

Comment:
This seems to be a duplicate on line 207, how about moving this to the "building" section?